src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -556,6 +556,19 @@ The following steps are performed:
 | `--namespace` `<string>`         | Namespace in which Submariner is installed (default `submariner-operator`)
 | `--yes`                          | Automatically answer yes to confirmation prompt
 
+### `recover-broker-info`
+
+`subctl recover-broker-info [flags]`
+
+This command recovers a lost `broker-info.subm` file.
+
+#### `recover-broker-info` flags
+
+| Flag                             | Description
+|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
```

Comment:
These are covered in https://github.com/submariner-io/submariner-website/pull/910/files#diff-0c05bfa14f9a42fbb974a98180586148ba00100482b3222225bc82ed53d7e2ddR35, I don't think the should be duplicated here.

Reply:
Although I see this is also duplicated for many commands. Can you raise an issue to de-dup this @Jaanki?

Reply:
Raised https://github.com/submariner-io/submariner-website/issues/914